includes/payment-method/class-card.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Class Sepa
+ *
+ * @package WCPay\Payment_Gateway
+ */
+
+namespace WCPay\Payment_Method;
+
+use WC_Payment_Gateway_WCPay;
+
+/**
+ * Credit Card Payment method.
+ * Right now behaves exactly like WC_Payment_Gateway_WCPay for max compatibility.
+ */
+class Card extends WC_Payment_Gateway_WCPay {
```

Comment:
Making the base class abstract would require more code changes in the unit tests and I was trying to keep this PR small but it's something that should probably be done eventually. I think when we add the 3rd payment method it will become more clear the structure that would work best.

Reply:
Agree, let's keep those as it is.